data/formats-data.js

```diff
@@ -6431,13 +6458,14 @@ exports.BattleFormatsData = {
 		tier: "RU",
 	},
 	jangmoo: {
+		randomBattleMoves: ["dragondance", "outrage", "dragonclaw", "earthquake", "substitute"],
 		tier: "LC",
 	},
 	hakamoo: {
 		tier: "NFE",
 	},
 	kommoo: {
-		randomBattleMoves: ["dragondance", "outrage", "closecombat", "poisonjab", "clangingscales"],
```

Comment:
clanging scales has potential for the z move clangorous soulblaze

Reply:
i mustve done something that messed with sync with recent changes. will fix